src/Symfony/Component/Notifier/Tests/Transport/NullTransportFactoryTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Tests\Transport;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
```

Comment:
bad namespace? this fails on travis

Reply:
It still fails, even with the correct namespace 🤔

Reply:
@nicolas-grekas Do you have any idea, what I did wrong?

Reply:
@jschaedl see https://github.com/symfony/symfony/pull/38842/files#r513755366

Reply:
Thank you @jderusse 👍